portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,8 +39,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         PrintWriter writer = response.getWriter();
 
         response.setContentType("text/html;");
```

Comment:
Looking at how you use the result on the client side, text/plain should be better contentType.